en/ros/mavros_offboard.md

```diff
@@ -2,19 +2,19 @@
 
 > **Caution** Offboard control is dangerous. If you are operating on a real vehicle be sure to have a way of gaining back manual control in case something goes wrong.
 
-The following tutorial will run through the basics of offboard control through mavros as applied to an Iris quadcopter simulated in Gazebo. At the end of the tutorial, you should see the same behaviour as in the video below, i.e. a slow takeoff to an altitude of 2 meters.
+The following tutorial will run through the basics of `OFFBOARD` control through MAVROS as applied to an Iris quadcopter simulated in Gazebo with SITL running. At the end of the tutorial, you should see the same behaviour as in the video below, i.e. a slow takeoff to an altitude of 2 meters.
```

Comment:
@hamishwillee The capitalised variant is the internal state machine representation. The user-facing documentation should have the flight modes like Offboard, Position, Altitude, Return, Land, Takeoff, etc. We should do a run through all the documentation and fix it everywhere. This would also make it consistent with QGC. 

![image](https://user-images.githubusercontent.com/5557676/27779788-ecd926f6-5fe2-11e7-9d1c-00a8611b31ca.png)